src/shared/containers/admin/advanced.jsx

```diff
@@ -125,81 +125,86 @@ class Advanced extends Component {
   render() {
     const emailServer = this.state.emailServerParams;
     const backend = this.state.backendParams;
+    const { user } = this.props;
     // const tunnelParams = this.state.tunnelParams || backend.tunnel || {};
     /* const hasTunnelParams = !!this.state.tunnelParams; */
     const saveBackendDisabled = !(
       this.state.backendParams || this.state.tunnelParams
     );
     const saveEmailDisabled = !this.state.emailServerParams;
-    return (
-      <Grid>
-        <Inner>
-          <Cell className="mdl-color--white" span={12}>
-            <div style={infoStyleD}>
-              Backend configuration
-              <Button
-                raised
-                disabled={saveBackendDisabled}
-                style={{ float: "right" }}
-                onClick={(e) => {
-                  e.preventDefault();
-                  this.onSaveBackend();
-                }}
-              >
-                SAVE
-              </Button>
+    const backendConfig = (
+      <Inner>
+        <Cell className="mdl-color--white" span={12}>
+          <div style={infoStyleD}>
+            Backend configuration
+            <Button
+              raised
+              disabled={saveBackendDisabled}
+              style={{ float: "right" }}
+              onClick={(e) => {
+                e.preventDefault();
+                this.onSaveBackend();
+              }}
+            >
+              SAVE
+            </Button>
+          </div>
+          <form style={infoStyleD}>
+            <div style={{ width: "520px" }}>
+              <TextField
+                onChange={() => {}}
+                label="Public Api url"
```

Comment:
Public Api could be displayed for owners but as read only

Reply:
done